test/integration/instance.test.js

```diff
@@ -1810,7 +1810,7 @@ describe(Support.getTestDialectTeaser('Instance'), function() {
       return this.ParanoidUser.create({ username: 'fnord' }).then(function() {
         return self.ParanoidUser.findAll().then(function(users) {
           return users[0].destroy().then(function() {
-            expect(users[0].deletedAt.getMonth).to.exist;
+            expect(users[0].deletedAt.val).to.be.equal('CURRENT_TIMESTAMP');
```

Comment:
Shouldn't this be populated with actual date object, which is returned by `CURRENT_TIMESTAMP` ?